api/db/changelog/db.changelog-166-add-access-module-tables.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="yonghao" id="changelog-166-add-access-module-tables">
+    <createTable tableName="access_module">
+      <column name="access_module_id" type="bigint" autoIncrement="true">
```

Comment:
Should the module "id" just be the primary key?

Reply:
It is the primary key now. I may not understand your question here.

Reply:
sorry, here I was actually referring to the module enum. That seems to effectively be the primary key - though it may be too nonstandard to use a varchar primary key here in our codebase. It is an unusual table though which is effectively a static set of rows.

Reply:
Tried to use enum as PK and seems it does not work well. See my below comments with Joel:
https://github.com/all-of-us/workbench/pull/5161#discussion_r658950497

Reply:
Got it, I wasn't sure - thanks for trying this out.